### PR TITLE
#1. Install all the dependencies in an easy way

### PR DESCRIPTION
**Pull request by guidotheelen**
Use Poetry as a dependency manager / virtual environment. This makes sure the user has all the dependencies with a single install command. Also deinstalling gets easier.

### CODE REVIEW

**Comment by guidotheelen** [comment]
Todo:
- Change version number to code
- Edit author to Google
- Add readme
- Remove fire for now

File: README.md [hunk 1]
```diff
@@ -4,14 +4,29 @@ Color and Black-and-White Noto emoji fonts, and tools for working with them.
...
-Then run make.  NotoColorEmoji is the default target.  It's suggested to use -j,
+* Step 1: Install Poetry.
+
+curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+
+And add it to your path:
+
+	`source $HOME/.poetry/env`
+
+* Step 2: Make sure the Python version defined in pyproject.toml (currently 2.7) or higher is available and use it.
```
Comment (RoelN) [inline README.md:15]: 2.7?

File: README.md [hunk 2]
```diff
@@ -4,14 +4,29 @@ Color and Black-and-White Noto emoji fonts, and tools for working with them.
...
 
-Building NotoColorEmoji currently requires a Python 2.x wide build.  To build
-the emoji font you will require a few files from nototools.  Clone a copy from
-https://github.com/googlei18n/nototools and either put it in your PYTHONPATH or
-use 'python setup.py develop' ('install' currently won't fully install all the
-data used by nototools).  You will also need fontTools, get it from
-https://github.com/behdad/fonttools.git.
-
-Then run make.  NotoColorEmoji is the default target.  It's suggested to use -j,
+* Step 1: Install Poetry.
```
Comment (RoelN) [inline README.md:7]: Perhaps also add a link to https://python-poetry.org/docs/#installation

File: README.md [hunk 3]
```diff
@@ -4,14 +4,29 @@ Color and Black-and-White Noto emoji fonts, and tools for working with them.
...
+
+curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
```
Comment (RoelN) [inline README.md:9]: Please put this inside `code` as well

File: README.md [hunk 4]
```diff
@@ -4,14 +4,29 @@ Color and Black-and-White Noto emoji fonts, and tools for working with them.
...
+
+And add it to your path:
+
+	`source $HOME/.poetry/env`
+
+* Step 2: Make sure the Python version defined in pyproject.toml (currently 2.7) or higher is available and use it.
+
+	`poetry env use 3.7.2`
+
+(If you don't have the correct python version installed, try using something like pyenv)
```
Comment (RoelN) [inline README.md:19]: python → Python. And how does using pyenv help?

File: README.md [hunk 5]
```diff
@@ -4,14 +4,29 @@ Color and Black-and-White Noto emoji fonts, and tools for working with them.
...
+
+* Step 3: Install project.
```
Comment (RoelN) [inline README.md:21]: "Install Python dependencies for this project"?

File: README.md (hunk 2 above)
Comment (RoelN) [reply README.md:7]: `* Step 1: ...` → `1. ...`

File: README.md (hunk 1 above)
Comment (guidotheelen) [reply README.md:15]: Made it 3.7.2 👍

File: README.md (hunk 2 above)
Comment (guidotheelen) [reply README.md]: ✅

File: README.md (hunk 3 above)
Comment (guidotheelen) [reply README.md]: Good one, lost them here for some reason 👍

File: README.md (hunk 5 above)
Comment (guidotheelen) [reply README.md]: ✅

**Comment by RoelN** [comment]
Currently fails on `libffi`:

image

I locally fix this with the following line, which is macOS-specific:

```bash
export PKG_CONFIG_PATH="$(brew --prefix libffi)/lib/pkgconfig/"
```

**Comment by davelab6** [comment]
Why use poetry instead of regular pip?

**Comment by guidotheelen** [comment]
Poetry gives some nice handles to manage dependancies. 
Example in pip: If you need dependancy A, you also need to install B. If you remove A again, B will still be in your in your project. 
But Rod also asked to switch over to pip, so I'll get on to it 🚀

**Pull request closed without merging**